test/classes/deinitializers/deinitAndDestruct.good

```diff
@@ -0,0 +1,3 @@
+deinitAndDestruct.chpl:11: error: ambiguous call 'C.deinit()'
+deinitAndDestruct.chpl:2: note: candidates are: C.~C()
+deinitAndDestruct.chpl:6: note:                 C.deinit()
```

Comment:
I'm not seeing the value in adding it.  It's not clear to me when a user would write this code and find the current message confusing, especially given that deconstructors aren't long for this world (i.e., should soon give a deprecation warning).